interfaces/builtin/desktop_calendar_service.go

```diff
@@ -0,0 +1,138 @@
+
+# Allow use of ObjectManager APIs, used to enumerate sources
+dbus (receive, send)
+	bus=session
+	interface=org.freedesktop.DBus.ObjectManager
+	path=/org/gnome/evolution/dataserver{,/**}
+	peer=(label=unconfined),
+
+# Allow access to properties on sources
+dbus (receive, send)
+	bus=session
+	interface=org.freedesktop.DBus.Properties
+	path=/org/gnome/evolution/dataserver/SourceManager{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.freedesktop.DBus.Properties
+	path=/org/gnome/evolution/dataserver/Calendar{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.freedesktop.DBus.Properties
+	path=/org/gnome/evolution/dataserver/CalendarFactory
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.freedesktop.DBus.Properties
+	path=/org/gnome/evolution/dataserver/CalendarView{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.freedesktop.DBus.Properties
+	path=/org/gnome/evolution/dataserver/Subprocess{,/**}
+	peer=(label=unconfined),
+# Allow access to methods
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.Calendar
+	path=/org/gnome/evolution/dataserver/{Subprocess,Calendar}{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.CalendarFactory
+	path=/org/gnome/evolution/dataserver/CalendarFactory
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.CalendarView
+	path=/org/gnome/evolution/dataserver/CalendarView{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.Source
+	path=/org/gnome/evolution/dataserver/SourceManager{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.Source.Removable
+	path=/org/gnome/evolution/dataserver/SourceManager{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.SourceManager
+	path=/org/gnome/evolution/dataserver/SourceManager
+	peer=(label=unconfined),
+
+# Allow clients to introspect the service
+dbus (send)
+	bus=session
+	interface=org.freedesktop.DBus.Introspectable
+	path=/org/gnome/evolution/dataserver/Calendar{,/**}
+	member=Introspect
+	peer=(label=unconfined),
+dbus (send)
+	bus=session
+	interface=org.freedesktop.DBus.Introspectable
+	path=/org/gnome/evolution/dataserver/CalendarFactory
+	member=Introspect
+	peer=(label=unconfined),
+dbus (send)
+	bus=session
+	interface=org.freedesktop.DBus.Introspectable
+	path=/org/gnome/evolution/dataserver/CalendarView{,/**}
+	member=Introspect
+	peer=(label=unconfined),
+dbus (send)
+	bus=session
+	interface=org.freedesktop.DBus.Introspectable
+	path=/org/gnome/evolution/dataserver/SourceManager{,/**}
+	member=Introspect
+	peer=(label=unconfined),
+`
+
+func init() {
+	registerIface(&commonInterface{
```

Comment:
Should we do what we do with the fuse interface and skip this if on Ubuntu 14.04?